models/Stream.cfc

```diff
@@ -787,6 +795,17 @@ component accessors="true" {
 		return variables.jStream.collect( variables.Collectors.toList() );
 	}
 
+	/**
+	 * A mutable reduction operation that accumulates input elements into a mutable result container, optionally transforming the accumulated result into a final representation after all input elements have been processed.
+	 * By default we will collect to a CFML array.
+	 *
+	 * This is a terminal operation.
+	 *
+	 */
+	function collectAsArray(){
+		return arraySlice( variables.jStream.collect( variables.Collectors.toList() ), 1 );
```

Comment:
Are you sure with 1 it works in Java arrays? and not 0?

Reply:
Yes, it does.  It is mostly used to convert the Java array to a CFML array.